README.md

```diff
@@ -2,59 +2,109 @@
-- An endpoint that shows the status of the file import process (a simple "running"/"finished"/... will do).
+- An endpoint that expects a payload containing a URL, which points to a very
+  large CSV file (multiple GBs) hosted somewhere. This endpoint should load the
+  data in that CSV file and save it to a local database.
+- An endpoint that shows the status of the file import process (a simple
+  "running"/"finished"/... will do).
 - An endpoint that allows cancelling a running import process.
 
-Your code, as well as any underlying dependencies, must be configured to run using a docker-compose file you must provide.
+Your code, as well as any underlying dependencies, must be configured to run
+using a docker-compose file you must provide.
 
-Please document in the code/config files any significant decisions you make as you go (be concise).
+Please document in the code/config files any significant decisions you make as
+you go (be concise).
 
 Tests are a plus.
 
 ## 2. Basic Requirements:
 
-These are the minimum things your solution must do, so we can properly evaluate your approach. We tried to keep it easy and simple, as to not take up much of your time:
+These are the minimum things your solution must do, so we can properly evaluate
+your approach. We tried to keep it easy and simple, as to not take up much of
+your time:
 
-- If you have used it before, use TypeScript for your solution. Solutions in vanilla JavaScript will also be accepted, but only if you haven't used TypeScript before.  
-- Use the same tools you would use in a production-grade project - it may be overkill for such a small app, but real-world apps are never this small.
-- Document how to start your application in a docker + docker-compose environment.
-- There is no need to create a web interface or worry about user authentication/authorization
+- If you have used it before, use TypeScript for your solution. Solutions in
+  vanilla JavaScript will also be accepted, but only if you haven't used
+  TypeScript before.
+- Use the same tools you would use in a production-grade project - it may be
+  overkill for such a small app, but real-world apps are never this small.
+- Document how to start your application in a docker + docker-compose
+  environment.
+- There is no need to create a web interface or worry about user
+  authentication/authorization
 
 ## 3. Optional Goals:
 
-Found the basic requirements too easy? Want to go the extra mile to impress us? Whatever the reason is, here are some optional ways in which you can give us a better sense of your approach, of your analytical and communication skills and of what you care about:
+Found the basic requirements too easy? Want to go the extra mile to impress us?
+Whatever the reason is, here are some optional ways in which you can give us a
+better sense of your approach, of your analytical and communication skills and
+of what you care about:
 
 - Add tests to your code.
```

Comment:
IMHO, I would add this as a requirement

Reply:
there's some context for this too - of course, I agree that "tests or didn't happen" (or "tests or GTFO"), but given the limited amount of effort we would feel comfortable asking from candidates, this was something that we thought we should sacrifice for the specific purpose of this challenge.

In practice some candidates in the past did add tests _because this helped them verify that what their code was doing did make sense_, which was good.

I'd recommend to keep this as is for now - and I would definitely encourage to ask about what they would test and how they would go about it, during the tech interview.

I've also been thinking about this - for example, would it make sense to provide a skeleton folder with something like a test coverage setup (so that submitting the solution would already show which coverage, if any, the project has), even some kinds of pre-made tests that verify this and that (a bit like acceptance tests in online code challenge platforms)...? On the other hand it may not be easy to do this without committing to specific things (for example, NodeJS so that a minimal `package.json` could be added, plus some CI steps...)... so I'm not sure about this, especially _at this stage_ - probably quite fun, but probably overkill (and probably if we wanted some of this we should evaluate SaaS platforms that do these things out of the box)... 😏